pkg/pillar/cmd/volumemgr/sizemgmt.go

```diff
@@ -40,16 +39,28 @@ func getRemainingDiskSpace(ctxPtr *volumemgrContext) (uint64, error) {
 				iterVolumeStatus.Key(), iterVolumeStatus.State)
 			continue
 		}
-		totalDiskSize += volumehandlers.GetVolumeHandler(log, ctxPtr, &iterVolumeStatus).UsageFromStatus()
+		reservedAppDiskUsage += volumehandlers.GetVolumeHandler(log, ctxPtr, &iterVolumeStatus).UsageFromStatus()
```

Comment:
I believe I know the answer, but I want to confirm. Is the disk usage by volumes that we are counting here overlapping with the disk usage counted for the corresponding content trees?

Reply:
We coykd the contenttree size on line 29 and the volumes here on line 42. Both of them use space independently (the CTs live in containerd).

Reply:
Okay, another question, for my understanding (I still have issues with the concept of a content tree). What we count for the content tree here is the size of the image stored in the CAS, yep? And the volume can use this image, but will it have its content copied? So the image can use the amount of storage twice its size: one for the content tree, one inside the volume?